### ISSUE TITLE
Consider content-descripition to determine attachment filename 

### ISSUE THREAD

**Issue by griesi007** [php-mime-mail-parser/php-mime-mail-parser#249]
I have an attachment where the filename is located in header "content-description". It would be great if the content-description header would also be evaluated to determine the filename. Currently this attachment gets the default filename "noname" which misses the suffix which is a problem
![15-07-_2019_09-35-42](https://user-images.githubusercontent.com/2854973/61201454-dbafda00-a6e4-11e9-9ffc-f8418b20ab4d.jpg)

**Comment by eXorus** [comment]
Currently we are using:
- disposition-filename
- content-name

Could you show me the value for these fields?

We could add the content-description but it's not a field that should handle the filename. So we will have to check if the value is a filename or a description, could be tricky.

**Comment by lucasvdh** [comment]
[RFC2183](https://tools.ietf.org/html/rfc2183) describes how a sender can provide information about content in the `Content-Disposition` header. This includes the filename. 

The `Content-Description` header is described as to provide "The ability to associate some descriptive information with a given body" and not to provide it's filename.

It wouldn't seem logical to me to include this behavior in this package. I've never encountered it and I reckon I've used this package to parse over 10 mil. emails now. If this is a case that you encounter alot I'd suggest just writing a fix outside of the package.

**TLDR;** sender is wrong.
